nixos/modules/config/zram.nix

```diff
@@ -92,6 +103,29 @@ in
           as there's no gain from keeping them in RAM.
         '';
       };
+
+      recommendedTuningSettings = lib.mkOption {
+        default = true;
```

Comment:
This should probably be `config.zramSwap.enable` for clarity.

Reply:
What do you mean? As I understand it, `recommendedTuningSettings` is only enabled, when `config.zramSwap.enable` is also enabled.

Reply:
Yes, but the documentation will currently say it is always enabled, because it does not have that context. We're not very consistent about this in nixpkgs, but it would be nice to improve.

Reply:
I see now.

Reply:
You haven't changed this, is it intentional?